src/Model/Post/UserNotification.php

```diff
@@ -52,6 +52,9 @@ class UserNotification
 	const TYPE_SHARED                 = 128;
 	const TYPE_FOLLOW                 = 256;
 	const TYPE_QUOTED                 = 512;
+	const TYPE_CONTACT_UNAVAILABLE    = 1024;
```

Comment:
These constants are meant for notification based on posts. I guess you should better have a look at how the contact related notifications are done, see for example the `FormattedNotification` classes.

Reply:
I'm finding this very confusing.  The only references to `FormattedNotification` I can see are notifications that the existing classes are deprecated.  Could you point me somewhere more specific, where such a notification is created?

I _think_ that a notification needs either a `UserNotification` type, which looks like a bad idea, or a verb, which would require publishing the new verb somewhere.  But then, using a shared verb seems weird when the status of a feed is purely an internal state.  Perhaps the intent is that notifications are supposed to show up in clients, and therefore a shared verb is needed?  I'd be happy with a "desktop-only" notification here, but maybe that's lazy.

I don't think I'm up for creating a whole new verb and injecting it into purl.org.  That sounds like something a responsible person would do 😅

Reply:
A verb would be wrong. Have a look at the addon `public_server`. I guess that you can use that kind of notification.